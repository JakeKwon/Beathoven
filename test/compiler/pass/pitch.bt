// pitch
pitch p = C4;
pitch q;
q = C4;
pitch r = 2;
string r_string = str_of_pitch(r);
print("This pitch is ", r_string);
