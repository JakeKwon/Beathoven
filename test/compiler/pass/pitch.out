This pitch is D4
